pkg/k8s/factory_functions.go

```diff
@@ -386,24 +387,39 @@ func ConvertToCNPWithStatus(obj interface{}) interface{} {
 		return &types.SlimCNP{
```

Comment:
Please create a new function dedicated for the conversion of `CiliumClusterwideNetworkPolicy` to `types.SlimpCNP`